algebird-core/src/main/scala/com/twitter/algebird/HyperLogLogSeries.scala

```diff
@@ -45,10 +92,10 @@ case class HLLSeries(bits: Int, rows: Vector[Map[Int, Long]]) {
     if (rows.isEmpty)
       SparseHLL(bits, Map())
     else
-      rows.zipWithIndex.map{
+      rows.iterator.zipWithIndex.map {
```

Comment:
why don't we want `sumOption` here rather than `reduce(_ + _)`? Or some logic that checks if there are more than `k` things to aggregate use `sumOption`?